nav2_map_server/CMakeLists.txt

```diff
@@ -40,6 +44,11 @@ add_executable(${map_saver_server_executable}
 add_executable(${costmap_filter_info_server_executable}
   src/costmap_filter_info/main.cpp)
 
+add_executable(${vector_object_server_executable}
```

Comment:
No component?

Reply:
Added composition node support in https://github.com/ros-planning/navigation2/pull/3930/commits/5cfde7ce3ff11deeeaef2b0d163a94fe95d44384